README.md

```diff
@@ -0,0 +1,133 @@
+# PoolTogether Governor Bravo Upgrade
+
+An upgrade to a "Bravo" compatible Governor for the PoolTogether DAO, built using the OpenZeppelin implemented and [Flexible Voting](https://github.com/ScopeLift/flexible-voting).
+
+#### Getting started
+
+Clone the repo
+
+```bash
+git clone git@github.com:ScopeLift/pooltogether-governor-upgrade.git
+cd pooltogether-governor-upgrade
+```
+
+Copy the `.env.template` file and populate it with values
+
+```bash
+cp .env.template .env
+# Open the .env file and add your values
+```
+
+```bash
+forge install
+forge build
+forge test
+```
+
+### Formatting
+
+Formatting is done via [scopelint](https://github.com/ScopeLift/scopelint). To install scopelint, run:
+
+```bash
+cargo install scopelint
+```
+
+#### Apply formatting
+
+```bash
+scopelint fmt
+```
+
+#### Check formatting
+
+```bash
+scopelint check
+```
+
+#### Scopelint spec compatibility
+
+Some tests will not show up when running `scopelint spec` because the methods they are testing are inherited in the `PoolTogetherGovernor`. In order to get an accurate picture of the tests with `scopelint spec` add an explicit `propose` method to the `PoolTogetherGoverno`r. It should look something like:
```

Comment:
```suggestion
Some tests will not show up when running `scopelint spec` because the methods they are testing are inherited in the `PoolTogetherGovernor`. In order to get an accurate picture of the tests with `scopelint spec` add an explicit `propose` method to the `PoolTogetherGovernor`. It should look like this:
```

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/20/commits/e857b73c3a2a48a7eb0049d5df1e57d52ca20194